### ISSUE TITLE
Remove toasts - they are likely inaccessible

### ISSUE THREAD

**Issue by nonprofittechy** [SuffolkLITLab/LITEFile#121]
https://adamsilver.io/blog/can-you-make-toast-messages-accessible/

Errors should be inline (and actually we are already using a mix of inline/toasts, so this should be more consistent)

**Comment by nonprofittechy** [comment]
I think most of these are gone already, but can do a further audit

**Comment by nonprofittechy** [comment]
@mnewsted — tagging for visibility on this Illinois feedback item.
